### PR TITLE
libcroco: Needs libxml2 on Linux

### PR DESCRIPTION
**Pull request by dholm**

### CODE REVIEW

File: Library/Formula/libcroco.rb [hunk 1]
```diff
@@ -14,6 +14,7 @@ class Libcroco < Formula
   depends_on "pkg-config" => :build
   depends_on "intltool" => :build
   depends_on "glib"
+  depends_on "libxml2" if OS.linux?
```
Comment (danabrand) [inline Library/Formula/libcroco.rb]: Thanks for your contribution. Please change this to
```ruby
depends_on "libxml2" unless OS.mac?
```

Comment (dholm) [inline Library/Formula/libcroco.rb]: Fixed!

**Comment by sjackman** [comment]
Fixes the error: `No package 'libxml-2.0' found`

**Comment by sjackman** [comment]
Thanks for your contribution, David.

**Pull request closed without merging**